envelope.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://mo-rise.github.com/brefv/envelope.json",
+    "type": "object",
+    "title": "Envelope",
+    "description": "An envelope including relevant timestamps, sender id and data type for any kind of message",
+    "examples": [
+      {
+        "sampled_at": 1632746168.503,
+        "sent_at": 1632746168.743,
+        "data": {},
+        "sender_id": "Bertil",
```

Comment:
Yep, I fully agree, the problems I had when thinking about this was several:
* MQTT clientId vs topic (`/OpenVTS/target/"identifier"/main_engine`) vs envelope sender_stamp
* What is a sender? Is it the sensor? The application that interfaces the sensor and sends the data to us? The target ship/drone/entity that houses the sensor?
* Should a sender_id be universally unique to avoid potential clashes between targets in OpenVTS/PONTOS? Probably? If so, who issues these?
* etc...